[2024-02-21T07:38:09.665+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-11T00:00:00+00:00 [queued]>
[2024-02-21T07:38:09.682+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-11T00:00:00+00:00 [queued]>
[2024-02-21T07:38:09.682+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:38:09.701+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-11 00:00:00+00:00
[2024-02-21T07:38:09.709+0000] {standard_task_runner.py:60} INFO - Started process 119 to run task
[2024-02-21T07:38:09.712+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-11T00:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpystuqlme']
[2024-02-21T07:38:09.715+0000] {standard_task_runner.py:88} INFO - Job 85: Subtask get_mail
[2024-02-21T07:38:09.774+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-11T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:38:09.865+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-11T00:00:00+00:00'
[2024-02-21T07:38:09.867+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:38:09.876+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240211T000000, start_date=20240221T073809, end_date=20240221T073809
[2024-02-21T07:38:09.888+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 85 for task get_mail (too many values to unpack (expected 2); 119)
[2024-02-21T07:38:09.928+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:38:09.944+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T07:59:48.621+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-11T00:00:00+00:00 [queued]>
[2024-02-21T07:59:48.628+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-11T00:00:00+00:00 [queued]>
[2024-02-21T07:59:48.628+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:59:48.637+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-11 00:00:00+00:00
[2024-02-21T07:59:48.641+0000] {standard_task_runner.py:60} INFO - Started process 388 to run task
[2024-02-21T07:59:48.643+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-11T00:00:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmp0ljlvap9']
[2024-02-21T07:59:48.645+0000] {standard_task_runner.py:88} INFO - Job 138: Subtask get_mail
[2024-02-21T07:59:48.673+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-11T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T07:59:48.728+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-11T00:00:00+00:00'
[2024-02-21T07:59:51.378+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T07:59:54.346+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T07:59:54.347+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-224.txt')
[2024-02-21T07:59:54.347+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.348+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T07:59:54.349+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-225.txt')
[2024-02-21T07:59:54.349+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.349+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T07:59:54.350+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-226.txt')
[2024-02-21T07:59:54.351+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.351+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T07:59:54.352+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-227.txt')
[2024-02-21T07:59:54.352+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.352+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T07:59:54.354+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-228.txt')
[2024-02-21T07:59:54.354+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.355+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T07:59:54.356+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-229.txt')
[2024-02-21T07:59:54.356+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.357+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T07:59:54.358+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-230.txt')
[2024-02-21T07:59:54.358+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.358+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T07:59:54.360+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-231.txt')
[2024-02-21T07:59:54.360+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.361+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T07:59:54.362+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-232.txt')
[2024-02-21T07:59:54.362+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.363+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T07:59:54.364+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-233.txt')
[2024-02-21T07:59:54.364+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:55.127+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T07:59:55.143+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240211T000000, start_date=20240221T075948, end_date=20240221T075955
[2024-02-21T07:59:55.229+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T07:59:55.249+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
